spec/instance_solr_doc_spec.rb

```diff
@@ -13,11 +13,11 @@
     let(:doc_hash) do
       isd.solr_doc_hash
     end
-    it 'is a Hash value' do
+    xit 'is a Hash value' do
```

Comment:
`xit` will produce lines like this when running rspec:

```
..***....*.****.....

Pending: (Failures listed here are expected and do not affect your suite's status)

  1) SparqlToSwSolr::InstanceSolrDoc#primary_triples retrieves BF2:subtitle property if it exists
     # Temporarily skipped with xit
     # ./spec/instance_solr_doc_spec.rb:118
```

I plan on making the tests work before the PR is done...